### PR TITLE
Dev jacques master script for users

### PR DESCRIPTION
**Pull request by Phillip-a-richmond**
Here we are adding an example directory to show users how to run FlexTyper. 

So far, we've implemented a ViralExample, which takes a fastq, indexes, and then searches for pathogen sequences. 

There are two example scripts, with different settings files on how the search is executed. 

Cheers,
Phil

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+This script relies upon these input files:
+- Fastq data (in FASTQ or FASTQ.GZ format)
+- Settings File (settings.ini), which defines search parameters
+- Query file (in .tsv/.txt format)
+- A master script to call the tool, located in scripts/ from main repository
+
+### Output files
+
+You will have these output files:
+- Output FM-index of reads: output\_0.fm9
```
Comment (Tixii) [inline examples/README.md:38]: output\_0.fm9 - This is the FM-Index generated for the read file

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- A master script to call the tool, located in scripts/ from main repository
+
+### Output files
+
+You will have these output files:
+- Output FM-index of reads: output\_0.fm9 
+- Output Results from Query: path\_query\_\_Results.tsv
+- Output matching reads: extracted\_reads.fa 
+
+There are additional files created during processing:
```
Comment (Tixii) [inline examples/README.md:42]: explain what each of these files are

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+- rc_MixedVirus_100.fasta
+- output_0
+- MixedVirus_100.fq
+- MixedVirus_100.fasta
+- fw_MixedVirus_100.fasta
+- indices.txt
+
+There is another script in this directory, which will run a separate set of indexing and querying with slightly different parameters. Before running, execute the clean-up function
```
Comment (Tixii) [inline examples/README.md]: what script?

File: examples/README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+```
+sh clean.sh
```
Comment (Tixii) [inline examples/README.md:55]: why do you use "sh" here but "bash" before?

File: examples/README.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+## Example 1 - Viral detection
+
+This is an example of ways to detect pathogenic virus reads from a raw fastq file. 
+
+This dataset includes a pre-made viral query file (pathogen\_query.txt), and a paired-end RNA-seq fastq file (Mixed\_virus\_1.fastq and Mixed\_virus\_2.fastq).
+
+~To generate your own query file, go to the fmformatter/ directory in the main repository.~ 
+
+It also includes two different execution scripts (ex1.sh and ex2.sh), which refer to different settings files (settings.ini, settings2.ini)
```
[the remark this answers is not in the document]
Comment (Tixii) [reply examples/README.md]: the names dont match... in the script below it says settings_2.ini 
image

File: examples/ViralExample/setting.ini [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+#             FlexTyper to perform a search
+#             The path can be absolute PATH's if needed.
+# 
+# AUTHOR      Godfrain Jacques Kounkou
+#---------------------------------------------------------
+
+queryFile 		= path_query.txt
+kmerSize 		= 30
+overlap 		= 0
+stride 			= 5
```
Comment (Tixii) [inline examples/ViralExample/setting.ini]: formatting

File: examples/ViralExample/setting_2.ini [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+#             FlexTyper to perform a search
+#             The path can be absolute PATH's if needed.
+# 
+# AUTHOR      Godfrain Jacques Kounkou
+#---------------------------------------------------------
+
+queryFile 		= path_query.txt
+kmerSize 		= 100
+overlap 		= 0
+stride 			= 5
```
Comment (Tixii) [inline examples/ViralExample/setting_2.ini:14]: formatting

**Review by Tixii — COMMENTED** [review]
Initial comments to be addressed

File: examples/README.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
+# Example of FlexTyper
+
+> This directory contains examples of how to run FlexTyper on small datasets.
+
+These examples will guide the usage of FlexTyper, and can be adapted for larger datasets in an applied setting.
+
+They incude ViralExample/ and WGSExample/
```
[the remark this answers is not in the document]
Comment (Phillip-a-richmond) [reply examples/README.md]: where?

[the remark this answers is not in the document]
Comment (Tixii) [reply examples/README.md]: incude

File: examples/README.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+
+## Example 1 - Viral detection
+
+This is an example of ways to detect pathogenic virus reads from a raw fastq file. 
+
+This dataset includes a pre-made viral query file (pathogen\_query.txt), and a paired-end RNA-seq fastq file (Mixed\_virus\_1.fastq and Mixed\_virus\_2.fastq).
```
[the remark this answers is not in the document]
Comment (Tixii) [reply examples/README.md]: why was this resolved? I dont see any changes?

**Review by Tixii — APPROVED** [review]

**Pull request merged**